index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Our First Tribute Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 id="title">Tribute</h1>
+    <p id="description">This is description paragraph!</p>
+    <form id="survey-form" action="#">
+
+        <label for="name">Please enter your name:</label>
+        <input id="name" type="text" required placeholder="Enter your name here:"><br>
+        <label for="email">Please enter your email:</label>
+        <input type="email" required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" id="email" 
+            name="email" placeholder="Enter your email here"><br>
+        <label for="reg">Your region</label><br>
+        <select name="regions" id="reg">
+            <option value="North_America">North America</option>
+            <option value="Sout_America">South America</option>
+            <option value="Asia">Asia</option>
+            <option value="Europe">Europe</option>
+            <option value="Africa">Africa</option>
+            <option value="Australia">Australia</option>
+            <option value="Antarctica">Antarctica</option>
+
+        </select><br>
```

Comment:
Please, try to be careful with blank lines and indentations. For readability, add two spaces of indentation. Do not use the tab key or you can change the tab key to make two spaces, not four 😊 .